src/plugins/yambi/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+include (LibAddMacros)
+
+if (DEPENDENCY_PHASE)
+	include (CheckIncludeFileCXX)
+	set (CMAKE_REQUIRED_QUIET ON)
+	check_include_file_cxx (codecvt HAVE_CODECVT)
+	if (NOT HAVE_CODECVT)
+		remove_plugin (yambi "because the current C++ library does not provide header file `codecvt`")
+		return ()
```

Comment:
That is unfortunate, since this style improves the readability of the `CMakeLists.txt` file. I moved the dependency checking code into a function in the latest commit.